client/my-sites/checkout/composite-checkout/composite-checkout.tsx

```diff
@@ -104,10 +104,14 @@ const wpcom = wp.undocumented();
 // otherwise we get `this is not defined` errors.
 const wpcomGetStoredCards = (): StoredCard[] => wpcom.getStoredCards();
 
-// Can be safely removed after 2021-03-03 when the FRESHPACK coupon expires
-import moment from 'moment';
+// Can be safely removed when we no longer need to rely on auto-applied coupons
+// for introductory new purchase pricing
```

Comment:
Could this whole block of code be moved to its own (typescript) file, maybe in `client/my-sites/checkout/composite-checkout/hooks`?  If it's going to be here a while, it would be nice to reduce the clutter in this file a bit.

Reply:
Absolutely! I'll make that change now.